bin/docker-build-policy-controller

```diff
@@ -13,11 +13,14 @@ bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
 . "$bindir"/_docker.sh
 # shellcheck source=_tag.sh
 . "$bindir"/_tag.sh
+# shellcheck source=_os.sh
+. "$bindir"/_os.sh
 
 if [ "$DOCKER_TARGET" = 'multi-arch' ]; then
     echo "DOCKER_TARGET may not be set to 'multi-arch' with $0" >&2
     exit 1
 fi
 
 dir=$( cd "$bindir"/../policy-controller && pwd )
-docker_build policy-controller "${TAG:-$(head_root_tag)}" "$dir/amd64.dockerfile"
+arch=$(architecture)
```

Comment:
TIOLI: could make this respect the ARCH variable if it's already set so that callers can override the arch.  Probably unnecessary...

Reply:
If the use case of overriding the architecture is ever needed we can add it then; for now I don't think that needs to be handled.